css/navbar.css

```diff
@@ -0,0 +1,11 @@
+nav.hide {
+  margin-top: -100px;
+  opacity: 0;
+  background-color: rgba(59, 59, 59, 0);
+}
+nav.show {
+  margin-top: 0px;
+  opacity: 1;
+  background: rgba(59, 59, 59, 0.7);
+  border-color: #444;
+}
```

Comment:
as i see you ignored animation . please check the next topic for it

https://gomakethings.com/controlling-the-transition-timing-of-show-and-hide-methods-with-vanilla-javascript/